sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/LifetimeAction.java

```diff
@@ -73,7 +73,7 @@ public Integer daysBeforeExpiry() {
      * @param daysBeforeExpiry The daysBeforeExpiry value to set
      * @return the Trigger object itself.
```

Comment:
this should return the LifetimeAction updated object?

Reply:
I see that these methods are present in both Trigger and LifetimeAction class?

Reply:
Fixed return statement.
Trigger class is internal.
LifeTimeAction is public.